packages/bs-reform/src/ReFormNext.re

```diff
@@ -38,7 +40,11 @@ module Make = (Config: Config) => {
     switch (validator) {
     | Validation.IntMin(field, min) => (
         Field(field),
-        Config.get(values, field) > min ? Valid : Error("Below minimum"),
+        Config.get(values, field) >= min ? Valid : Error("Below minimum"),
```

Comment:
You can use Yup's error message, which include the `min` value https://github.com/jquense/yup/blob/master/src/locale.js#L38

Reply:
noice, really soon we are going to extract all these and put in a real errors dictionary so users are able to inject their own.